### PR TITLE
new header again

### PR DESCRIPTION
**Pull request by mhasilau**
image
image

### CODE REVIEW

**Review by PaulRomanov — APPROVED** [review]
looks kosher

File: src/app/app.module.ts [hunk 1]
```diff
@@ -2,20 +2,23 @@ import { NgModule } from '@angular/core';
...
 
 import { ToastrModule } from 'ngx-toastr';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { HeaderComponent } from './components/header/header.component';
 
 @NgModule({
-  declarations: [AppComponent],
+  declarations: [AppComponent, HeaderComponent],
```
Comment (rastorc3v) [inline src/app/app.module.ts:14]: Create module for header please

File: src/app/components/header/header.component.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<div class="header">
```
Comment (rastorc3v) [inline src/app/components/header/header.component.html:1]: Still div with class 'header'. (should be <header></header>

File: src/app/app.module.ts (hunk 1 above)
Comment (rastorc3v) [reply src/app/app.module.ts:14]: Can't find header.module.ts here

**Pull request merged**